src/styles.css

```diff
@@ -26,7 +26,6 @@
 }
 
 .btn:hover {
-    border: 1px solid white;
```

Comment:
We should ceep a border when hovering on button or remove the border from the button when not hovering
![image](https://github.com/pib-rocks/cerebra/assets/144774775/b5f06a5a-be7b-4d80-a6e8-5cb72febab37)
![image](https://github.com/pib-rocks/cerebra/assets/144774775/4bd99ca0-e81f-4493-a871-1768be85ed8a)